plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -215,6 +215,12 @@ public WriteMapping toWriteMapping(ConnectorSession session, Type type)
         return delegate.toWriteMapping(session, type);
```

Comment:
Can we re-word the commit as `Allow jdbc connector to automatically coerce type during table creation`